pkg/controllers/genericissuer_controller.go

```diff
@@ -153,3 +124,40 @@ func validateIssuer(spec *api.AWSPCAIssuerSpec) error {
 	}
 	return nil
 }
+
+func (r *GenericIssuerReconciler) getConfig(ctx context.Context, spec *api.AWSPCAIssuerSpec) (aws.Config, error) {
+	if spec.Region != "" {
+		return config.LoadDefaultConfig(ctx,
+			config.WithRegion("us-east-1"),
```

Comment:
Will hard-coding `us-east-1` here break current users of this package who don't specify a region in their spec but don't run in default `us-east-1`?